packages/gatsby-transformer-remark/src/__tests__/extend-node.js

```diff
@@ -85,7 +85,13 @@ Where oh where is my little pony?
           types: [{ name: `MarkdownRemark` }],
         }
       ).then(result => {
-        expect(_.isString(result.data.listNode.excerpt))
+        try {
```

Comment:
What's the value of wrapping in a `try/catch` here?

If we're grabbing a deeply nested property, I'd rather just use `_.get` or something!


i.e.

```js
const excerpt = _.get(result, 'data.listNode[0].excerpt');

expect(excerpt).toEqual(expect.any(String));

// or as Michal said

expect(excerpt).toMatchSnapshot();
```

It also seems slightly odd to use `async/await` and the callback (`done`) variant for this test!

Reply:
well it seems that just expect does not work here. Even if the test fails and we get the error in the console the test passes. this is the reason i surrounded it with a try catch so that i could catch the error and fail the test accordingly. This is actually done in the other test module.

Reply:
we just finished pairing session - can confirm problem that without it test passes and after get console output that expected value don't match:
```
 PASS  packages/gatsby-transformer-remark/src/__tests__/extend-node.js
  Excerpt is generated correctly from schema
    ✓ correctly loads a default excerpt (20ms)

Test Suites: 1 passed, 1 total
Tests:       1 passed, 1 total
Snapshots:   0 total
Time:        1.342s
Ran all test suites matching /packages\/gatsby-transformer-remark\/src\/__tests__\/extend-node.js/i.
(node:33401) UnhandledPromiseRejectionWarning: Error: expect(received).toEqual(expected)

Expected value to equal:
  false
Received:
  true
```

Reply:
I guess to work around this we would need to create explicit `new Promise`, return it at the end of test and resolve it in handler function where we do assertions, but using `done` is fine too